yolo.py

```diff
@@ -155,11 +155,11 @@ def get_os_friendly_name():
 
 print("Command: " + colored(res_command, 'blue'))
 if yolo_safety_switch == True or ask_flag == True:
-  print("Execute the command? Y/n ==> ", end = '')
+  print("Execute the command? Y/n (default Y) ==> ", end = '')
```

Comment:
Ha, I also just made changes to this file :) - but I'll accept yours first. Will ask a couple more users about this also. Thanks!